clang/include/clang/StaticAnalyzer/Core/PathSensitive/SymbolManager.h

```diff
@@ -707,6 +678,19 @@ class SymbolVisitor {
   virtual bool VisitMemRegion(const MemRegion *) { return true; }
 };
 
+template <typename T, typename... Args>
+const T *SymbolManager::acquire(Args &&...args) {
+  llvm::FoldingSetNodeID profile;
+  T::Profile(profile, std::forward<Args>(args)...);
```

Comment:
You can't forward the args here because that would mean they may become moved-from.
If they could become moved-from, then the next forward would/could become a bug.

```suggestion
  T::Profile(profile, args...);
```